worker/runc/runc_test.go

```diff
@@ -56,7 +56,7 @@ func TestRuncWorker(t *testing.T) {
 	w, err := base.NewWorker(workerOpt)
 	require.NoError(t, err)
 
-	img, err := source.NewImageIdentifier("docker.io/library/busybox:latest")
+	img, err := source.NewImageIdentifier("docker.io/library/alpine:latest")
```

Comment:
why?

Reply:
`busybox` do not /run folder by default, so modified to `alpine`

Reply:
Let's mkdir then